source/conf.py

```diff
@@ -296,6 +296,7 @@ def ultimateReplace(app, docname, source):
 ultimate_replacements = {
     "{{{release}}}" : release,
     "{{{apiversion}}}" : release.replace(".", "-", 2),
+    "{{{mongodb42recommended}}}" : mongodb42recommended,
```

Comment:
since now we support 4.4 - we should probably change this to `mongodb44recommended` since that is highest supported mongodb version

Reply:
Currently, all our install and configuration instructions rely on 4.2. Maybe it will be better to change them to 4.4 later. This PR will update the 1.6 release docs, and I think it's better to avoid additional changes in the installation docs without additional testing.

Reply:
ok, but it's a good question for Sergey about which version should be default for everything, so let's first ask him